docs/marketplace/dev/feature-walkthroughs/202108.0/merchant-category-feature-walkthrough.md

```diff
@@ -5,14 +5,30 @@ description: Merchant categories allows grouping merchants by categories.
 template: concept-topic-template
 ---
 
-The *Merchant Category* feature allows grouping merchants by categories.
+The *Merchant Category* feature allows splitting merchants into various categories in order to extend business logic of the project.
 
 {% info_block warningBox "User documentation" %}
 
 To learn more about the feature and to find out how end users use it, see [Merchant Category feature overview](/docs/marketplace/user/features/{{page.version}}/merchant-category-feature-overview.html) for business users.
 
 {% endinfo_block %}
 
+## Module dependency graph
+
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/19aac040-a607-4a20-8edf-a81473e293e9.png?utm_medium=live&utm_source=custom)
```

Comment:
Please update the diagram and list of modules according to the latest agreements. @az-joss